internal/cli/quickstarts.go

```diff
@@ -123,6 +134,11 @@ func downloadQuickstart(cli *cli) *cobra.Command {
 			}
 
 			cli.renderer.Infof("Quickstart sample sucessfully downloaded at %s", target)
+
+			qsType := quickstartsTypeFor(client.GetAppType())
+			if err := promptDefaultURLs(context.TODO(), cli, client, qsType); err != nil {
```

Comment:
I missed this on the first review, sorry 🙏
```suggestion
			if err := promptDefaultURLs(cmd.Context(), cli, client, qsType); err != nil {
```